### DISCUSSION TITLE
Why is stdout_callback=yaml not supported in the Universal container image?

### DISCUSSION THREAD

**Discussion by adietrich-ussignal** [aristanetworks/avd#3460] [Q&A] [answered]
While converting from the avd-all-in-one-container to the new Universal image, I noticed a failure with the following message:
`ERROR! Invalid callback for stdout specified: yaml`

After commenting out `stdout_callback=yaml` in the ansible.cfg file, I had no issues with the universal container. It may be helpful to note the requirement if its not already documented in the usage of the universal container.

**Comment by carlbuchmann** [comment]
@adietrich-ussignal - Thank you for testing and bringing this to our attention. In the avd-all-in-one-container we installed the `community.general` collection -> https://github.com/arista-netdevops-community/avd-all-in-one-container/blob/24d2ddd0fd9667488aed59ec9b14ba397d0bb6dd/Dockerfile#L91.
To workaround this issue, you can add a step in your CI workflow to install the `community.general` collection.

@ankudinov @ClausHolbechArista - should we consider adding support for this in the universal container as well? IMO, It makes sense as it can make error messages more readable.

**Comment by ankudinov** [comment, marked as answer]
We'll add `community.general` as part of #3474  
It is possible that we have to reconsider that in future once we introduce security scanning, etc.
